src/test/regress/sql/reassign_owned.sql

```diff
@@ -0,0 +1,140 @@
+CREATE ROLE distributed_source_role1;
+create ROLE "distributed_source_role-\!";
+
+CREATE ROLE distributed_target_role1;
+
+set citus.enable_create_role_propagation to off;
+create ROLE local_target_role1;
+alter role local_target_role1 SET work_mem = '64MB';
+
+\c - - - :worker_1_port
+
+CREATE ROLE local_target_role1;
```

Comment:
Let's really make local_target_role1 a local role, right now it's distributed as far as I see in the test output.

```suggestion
set citus.enable_create_role_propagation to off;
CREATE ROLE local_target_role1;
```